src/calculateRentalCost.js

```diff
@@ -20,7 +20,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  if (days >= 7) {
+    return cost * days - 50;
+  } else if (days >= 3) {
+    return cost * days - 20;
+  } else {
```

Comment:
no need for this else
```suggestion
  }
```